examples/msg_sender/src/main.sw

```diff
@@ -0,0 +1,28 @@
+contract;
+
+use std::{
+    address::Address, 
+    assert::assert,
+    chain::auth::{AuthError, Sender, msg_sender}, 
+    panic::panic, 
+    result::*
+};
+
+abi MyOwnedContract {
+    fn receive(field_1: u64) -> bool;
+}
+
+const OWNER:b256 = 0x9ae5b658754e096e4d681c548daf46354495a437cc61492599e33fc64dcdc30c;
```

Comment:
soonish (tm) we'll be able to do
`const OWNER = ~Address::from(0x9ae5b658754e096e4d681c548daf46354495a437cc61492599e33fc64dcdc30c);`
and then below:
`assert(addr == OWNER);`
but the way you have it works with the current state of things.